scripts/flash

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "Getting mac address from device..."
+mac=$(esptool.py read_mac | grep -m 1 'MAC:' | grep -o '.\{8\}$' )
+mac=${mac//:/}
+
+echo "MAC ending: ${mac}"
+
+sed -i "s/devicename: tagreader/devicename: tagreader_${mac}/" tagreader.yaml
```

Comment:
Let's use the new substitutions command line option

```
esphome -s mac $mac tagreader.yaml compile
```

Reply:
Actually instead of substituting mac, we should just override `$devicename` so that `tagreader.yaml` will still work without substitutions from command line

Reply:
I made a commit for this and didn't push it..haha

Reply:
Pushed now

Reply:
https://esphome.io/guides/configuration-types.html#command-line-substitutions